docs/source/contributing/testing.rst

```diff
@@ -1,21 +1,24 @@
 ============
 Adding Tests
 ============
-When adding a new feature, it should always be tested. Tests prevent
-manim from breaking at each new feature added by checking if any other
+If you are adding new features to manim, you should add appropriate tests for them. Tests prevent
+manim from breaking at each change by checking that no other
 feature has been broken and/or been unintentionally modified.
 
 How Manim Tests
 ---------------
 
-To conduct our tests, we use ``pytest``. Running ``pytest`` in the root of
+Manim uses ``pytest`` as its testing framework. Running ``pytest`` in the root directory of
 the project will start the testing process, and will show if there is
 something wrong.
```

Comment:
> Manim uses ``pytest`` as its testing framework. To start the testing process, go to the root directory of the project and run ``pytest`` in your terminal. Any errors that occur during testing will be displayed at your convenience.

Perhaps something like this sounds more beginner friendly for new contributors

Reply:
Agreed